src/main/kotlin/com/terraformation/backend/email/EmailService.kt

```diff
@@ -109,6 +109,34 @@ class EmailService(
     }
   }
 
+  /**
+   * Sends an email notification to all users.
+   *
+   * @param [model] Model object containing values that can be referenced by the template.
+   * @param [requireOptIn] If false, send the notification even if the user has not opted into email
+   *   notifications. The default is to obey the user's notification preference, which is the
+   *   correct thing to do in the majority of cases.
+   */
+  fun sendAllUsersNotification(
+      model: EmailTemplateModel,
+      requireOptIn: Boolean = true,
+  ) {
+    val allUsers = userStore.fetchUsers(requireOptIn)
+    allUsers.forEach { user ->
+      try {
+        if (requireOptIn && !user.emailNotificationsEnabled) {
+          log.info(
+              "Skipping email notification for user ${user.userId} because they didn't enable it")
```

Comment:
This would be a sign of a bug, right? `fetchUsers()` should already be filtering these users out.

Reply:
Yes, I will remove this condition, it's not necessary